src/org/openj9/envInfo/MachineInfo.java

```diff
@@ -22,6 +22,9 @@
 import java.util.LinkedHashMap;
 import java.util.Map;
 import java.lang.management.ManagementFactory;
+import java.util.HashMap;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
```

Comment:
You can remove these import for this PR.

Reply:
removed👍🏼